src/Internal/Test/E2E/Wallets.purs

```diff
@@ -169,10 +169,13 @@ namiConfirmAccess extId re =
 
```

Comment:
inWalletPageOptional must be used in every `*confirmAccess` to account for the case where the access is already granted.

Reply:
See how it is used for eternl